nbclient/client.py

```diff
@@ -323,7 +323,7 @@ def reset_execution_trackers(self):
         self.output_hook_stack = collections.defaultdict(list)
         # our front-end mimicing Output widgets
         self.comm_objects = {}
-        self.state_history = []
+        self.state_history = [ExecutionState.IDLE, timestamp()]
```

Comment:
It must be `self.state_history = [(ExecutionState.IDLE, timestamp())]`. Maybe we should add a basic test?

Reply:
hah I noticed this immediately after pushing. I will add some state checking to tests